spring-cloud-gcp-starters/spring-cloud-gcp-starter-core/src/main/java/org/springframework/cloud/gcp/core/GCPProperties.java

```diff
@@ -27,7 +28,7 @@
 
 	private String projectId;
 
-	private String jsonKeyFile;
+	private Resource credentialKeyFile;
```

Comment:
@viniciusccarvalho Make sense. I just wasn't sure if using `GcpProperties.projectId` outside of auto-configuration machinery is ok, as in @joaoandremartins example. I guess the answer is yes.